test/onnx/test_pytorch_onnx_shape_inference.py

```diff
@@ -268,6 +272,74 @@ def test_resize_after_concat(self):
         )
         self.run_test(g, resize.node(), expect_tensor("Float", shape=(4, 32, 128, 128)))
 
+    def test_maintain_dynamic_shapes_of_unreliable_nodes(self):
```

Comment:
I'm sorry this seems to be back and forth. The test in this current format is more suited in `test_pytorch_onnx_no_runtime.py`. In my original comment of checking out `test_pytorch_onnx_shape_inference.py`, it was meant for creating another test that adheres to the style of other test cases in that file: constructing an torch onnx IR directly, and validate shape inference. This process does not go through `export` api, and is a bit more scoped in on the shape inference calls.

Reply:
I see. I was wondering the same thing that this test case seems doesn't fit to `test_pytorch_onnx_shape_inference.py`. I will move it to no_runtime then.

Reply:
Done